docs/Manual.md

```diff
@@ -0,0 +1,260 @@
+
+#### My Location
+  - Announces the user's location.
+
+#### Around Me
+  - Announces the areas around the user.
+
+#### Ahead of Me
+  - Announces things that are directly in front of the user.
+
+#### Nearby Markers
+  - Announces the markers that are nearby
+  
+## Sidebar
+
+#### Head Tracking Headphones
+- Gives user two options to choose from: ```Check Your Headphones``` & ```Forget This Device```
+- ```Check Your Headphones```
+  - Creates an audio beacon to the user's right to test if the headphones are working
+  - If the user turns his head to the beacon then a ding sound is played
+  - After testing it presents a button to allow the user to return back to homepage
+- ```Forget This Device```
+  - Disconnects the user's headphones from soundscape
+  - Headphones can be reconnected after
+
+#### Recreational Activities
+- Currently In Development
+- Organizations can create their own soundscape-based experiences
+
+#### Settings
+##### General Settings
+  - ```Language & Region```
+    - Changes your language and units of measurements(imperial or metric)
+  - ```Voice```
+    - Change the Speaking rate and voice of operator
+  - ```Audio Beacon```
+    - Allows the user to enable ```Departure and Arrival Cues```
+    - Change the sound (audio style) of the audio beacon
+    - Enable Haptic Styles (Wand or Pulse)
+      - Vibrate phone
+  - ```Volume```
+    - Alows you to change the voTlume of the beacon, the voice, and the sound effects using slidebars
+  - ```Head Tracking Headphones```
+    - Works the same as the head tracking button seen in the waffle button
+  - ```Siri Shortcuts(Beta)```
+    - Lists addable shortcuts for the users to say to siri to get information
+    - Examples include: "What's My Location?", "What's Around Me?", "Hear Markers Nearby", etc.
+
+##### Media Controls
+  - Allows user to control Soundscape with their headphone buttons and with the media controls on the home page of their phone(lets the user pause soundscape)
+  - If media controls are active the user can not use other volumetric-based media like Spotify
+
+##### Manage Callouts
+  - Manages the information that will be called out to the user
+  - Examples include: Allow Callouts, Places and Landmarks, Mobility, Distance to the Audio Beacon, and Repeat Callouts
+
+##### Soundscape Street Preview
+  - Allows the user to include unnamed roads in the Soundscape Street Preview
+
+##### Troubleshooting
+- Allows the user to see how accurate Soundscape sees their location in terms of a by foot radius
+- Offers a button to check the users audio
+- Lets the user clear all their stored map data
+
+##### About
+- Provides a button that:
+  - Shows the current version updates
+  - The Privacy Policy
+  - The Services Agreement
+  - The Copyright Notices
+  - The Youtube Channel
+
+##### Telemetry
+- Gives the user the option to stop sharing their Soundscape usage data
```

Comment:
Note that this doesn't currently have any effect, as no usage data is collected either way.